README.md

```diff
@@ -15,13 +15,24 @@ The (internal) project-board is [here](https://github.com/orgs/hpi-sam/projects/
 5. Run `npm run setup` from the root folder
 6. (Optional) We have a list of recommended [vscode](https://code.visualstudio.com/) extensions. We strongly recommend you to use them if you are developing. You can see them via [the `@recommended` filter in the extensions panel](https://code.visualstudio.com/docs/editor/extension-marketplace#_recommended-extensions).
 
+## Setup - Database
+
+Make sure to have a [PostgreSQL](https://www.postgresql.org/) (version 14) server running. It must use a configuration according to the environment variables specified in `backend/.env` for name, user, password, host and port.
+We provide a `docker-compose` service named `db` for this (see also [below](#with-docker-compose-recommended)). You can start this service by using `docker-compose up -d db`.
+
+Before starting the application make sure that you have run the migrations (run `npm run migration:run` in `./backend`).
```

Comment:
```md
(run `npm run migration:run` in `./backend`)
```

This is not correct for windows. Maybe just reference the appropiate section in the backend README instead.